countNumberOfLowercase.js

```diff
@@ -0,0 +1,13 @@
+export const typeOfArgumentErrorMessage = "There should be a string!"
+
+export function countNumberOfLowercase(string) {
+    let count = 0;
+    if (typeof string != "string") { throw new Error(typeOfArgumentErrorMessage) } else {
+        for (let i = 0; i < string.length; i++) {
+            if (string[i] >= 'a' && string[i] <= 'z') {
```

Comment:
would this solution be better and more readable if you used a regular expression?

Reply:
if you don't know what it is and how to use it, I have a youtube video about it:)

Reply:
done